src/build/stable/utils/versions/rust.ts

```diff
@@ -0,0 +1,22 @@
+import { pathToFileURL } from 'url';
+
+import { execSyncPretty } from '../exec_sync_pretty';
+
+export function getRustVersionLocal(): string {
```

Comment:
Shouldn't we use the more readable naming convention: `getLocalRustVersion`? And do we need to say local?

Reply:
done